client/src/modules/stock/movements/modals/search.modal.html

```diff
@@ -12,80 +12,100 @@
   </div>
 
   <div class="modal-body">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+        <div class="tab-body">
+          <!-- movements  -->
+          <div class="form-group">
```

Comment:
You actually don't need the `class="form-group"` any more.  Bootstrap mostly uses this to organised validation and associate `<label>`s and `<input>`s.